src/cli/commands/list.rs

```diff
@@ -21,18 +20,16 @@ impl GenericCommand for ListCommand {
                 let bg = &nc.color;
                 let fg = bg.text_color();
                 writeln!(
-                    out,
+                    out.handle,
                     "{}",
                     config
                         .brush
                         .paint(format!(" {:24}", nc.name), fg.ansi_style().on(bg))
                 )?;
             }
         } else {
-            let stdout = io::stdout();
-            let mut out = stdout.lock();
```

Comment:
good that this can be removed. I think this was a leftover from a previous time where I didn't have access to `out` in the commands... not sure.